cinema_project/login/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from . import views
+
+
+urlpatterns = [
```

Comment:
If you separated auth logic from other app logic, `login` is not really a fit name for the app. You could try `accounts`, `auth` or something else that describes the whole app logic, not just the login functionality.